include get_utxo_info and delete_wallet (#87)
witnet/documentation: docs/developer/wallet-api.md

  - `address`: *String*, the recipient address.
  - `amount`: *number*, value to transfer in nanoWits.
  - `time_lock`: *number*, indicates the epoch from which the data request could run before, before this epoch the request is ignored.

> - `utxo_strategy` (optional): *UtxoSelectionStrategy*, enum that allows to choose 3 different strategies, BigFirst, SmallFirst and Random.

Reviewer: ```suggestion
- `utxo_strategy` (optional): *`random` | `big_first` | `small_first`*, enum that allows to choose 3 different strategies for selecting which UTXOs are used as input of the VTT transaction.
```

I got this:
```json

{
  "jsonrpc": "2.0",
  "error": {
    "code": -32602,
    "message": "Invalid params: unknown variant `bigfirst`, expected one of `random`, `big_first`, `small_first`.",
    "data": {
      "schema": "https://github.com/witnet/witnet-rust/wiki/Create-Vtt"
    }
  },
  "id": 1
}
```

Reviewer: I forgot the expand the UTXO acronym

> - `selected_utxos` (optional): *HashSet<OutputPointer>*, output pointers selected to be spent.

Reviewer: ```suggestion
- `selected_utxos` (optional): *Array*, array of selected output pointers to be spent.
```

Example:

> 

Reviewer: ```json
{
  "jsonrpc": "2.0",
  "method": "create_vtt",
  "params": {
    "session_id": "e02d345f545d9516c86b3c5ae40c29be249d546f255e34e082c98599a946e173",
    "wallet_id": "8f5b85981addad621a86f01a1ddb646ccd90620c95247948ce8d99feefd0496c",
    "fee": 1,
    "label": "",
    "outputs": [
      {
        "address": "twit1nfkythqds4r2hz3le2zaauxtl7yum76jd0ut9c",
        "amount": 10000000001,
        "time_lock": 0
      }
    ],
    "utxo_strategy": "random",
    "selected_utxos": [
      "3a1ff5de49142d21280dc62dbdc02810a9665e9c23bb27b9094d6dbd6f4e527f:0",
      "3cebaae8cc942056c475d7f9866a5de1be41aea0a1bcdb63edd533b9caef4043:0"
    ]
  },
  "id": 1
}
```

```

### delete_wallet

> The JsonRPC method `delete_wallet` is used to delete the current wallet.

Reviewer: ```suggestion
The JsonRPC method `delete_wallet` is used to delete the wallet with the specified ID. It also removes the current active session. 
```

```

### get_utxo_info

> The JsonRPC method `get_utxo_info` is used to query the current utxos for a given wallet.

Reviewer: ```suggestion
The JsonRPC method `get_utxo_info` is used to query the current UTXOs for a given wallet.
```

Reviewer: Maybe expand the acronym too?

Request with parameters:

- `session_id`: *String*, session ID assigned to you when you unlocked the wallet. See [unlock_wallet](#unlock_wallet).
- `wallet_id`: *String*, ID associated to the wallet. See [get_wallet_infos](#get-wallet-infos).

```json
{
	"jsonrpc": "2.0",
	"method": "get_utxo_info",
  "params": {
    "session_id": "9fa1d779afea88a29768dd05647e37b2f64fc103c1081b0ee9e62fb283f5cd02",
    "wallet_id": "6c344625884c2f910065ab170dc18ad3cbbc03c7234507c7c22dbd78e3b26667"
  },
  "id": "1"
}
```

> Response with a HashMap with all the information about the utxos of a specified wallet:

Reviewer: ```suggestion
Response with a HashMap with all the information about the UTXOs of a specified wallet:
```

> - `output_info[]`: *HashMap<String, OutputInfo>*, information related of all the utxos (amount, pkh and time_lock)

Reviewer: ```suggestion
- `output_info[]`: *HashMap<String, OutputInfo>*, information related of all the utxos (`amount`, `pkh` and `time_lock`)
```
